subiquity/server/autoinstall.py

```diff
@@ -24,8 +25,13 @@ class AutoinstallError(Exception):
 class AutoinstallValidationError(AutoinstallError):
     def __init__(
         self,
-        owner: str,
+        section: str,
+        message: Optional[str] = None,
     ):
-        self.message = f"Malformed autoinstall in {owner!r} section"
-        self.owner = owner
+        if not message:
```

Comment:
A standard across Subiquity is to prefer "is not None" over this case.  I suspect that's a little better here.